code/modules/mob/living/blood.dm

```diff
@@ -85,6 +85,9 @@
 	if(!(NOBLOOD in dna.species.species_traits))
 		..()
 
+	if(HAS_TRAIT(src, TRAIT_NOPULSE)) // Fulpstation Bloodsuckers edit: They don't need to be here
```

Comment:
will this run ..() twice?

Reply:
also, are you sure that you want to have species that aren't supposed to have blood bleed here? it could cause some weirdness.